util/analyze_output_utils.go

```diff
@@ -216,3 +216,55 @@ func (r FileAnalyzeResult) OutputText(analyzeType string, format string) error {
 	}
 	return TemplateOutputFromFormat(strResult, "FileAnalyze", format)
 }
+
+type FileLayerAnalyzeResult AnalyzeResult
+
+func (r FileLayerAnalyzeResult) OutputStruct() interface{} {
+	analysis, valid := r.Analysis.([][]util.DirectoryEntry)
+	if !valid {
+		logrus.Error("Unexpected structure of Analysis.  Should be of type []DirectoryEntry")
+		return errors.New("Could not output FileAnalyzer analysis result")
```

Comment:
I think this error will get outputted somewhere further up the stack right? Can you just do an `errors.Wrapf` here and return that

Reply:
Sorry, what error do you mean? There isn't a specific error to wrap, which is why I think all the AnalyzeResults are returning new ones.

Reply:
Ah sorry, I meant to combine these errors into one and return them without logging anything here. I see that you're just following the pattern from what was already here though, so it's fine for now. We should clean this error handling up later.